test/support/isolated_unit.rb

```diff
@@ -72,6 +72,7 @@ def make_basic_app
       end
       def app.name; 'IsolatedRailsApp'; end # rubocop:disable Style/SingleLineMethods
       app.respond_to?(:secrets) && app.secrets.secret_key_base = '3b7cd727ee24e8444053437c36cc66c4'
+      app.respond_to?(:credentials) && app.credentials.secret_key_base = '3b7cd727ee24e8444053437c36cc66c4'
```

Comment:
can we extract this secret into a local var or constant and use in both these places?